src/index.html

```diff
@@ -15,8 +15,108 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    >
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    >
+    <title>Moyo header</title>
```

Comment:
There are two `title` tags within the `head` section. Remove the duplicate `title` tag to comply with proper HTML structure.